go/teams/member_test.go

```diff
@@ -506,6 +506,53 @@ func TestMemberAddNoKeys(t *testing.T) {
 	}
 }
 
+func TestMemberDetailsResetAndDeletedUser(t *testing.T) {
+	tc, owner, otherA, otherB, name := memberSetupMultiple(t)
+	defer tc.Cleanup()
+
+	tc.G.UIDMapper.SetTestingNoCachingMode(true)
```

Comment:
is this ok or should we be doing something else?

Reply:
looks good, hopefully this isn't cleared by `kbtest.Logout` somehow